Automation/Automation/Pages/AccountPage.cs

```diff
@@ -8,12 +9,39 @@ public class AccountPage
         #region Selectors
 
         private readonly By _welcomeText = By.CssSelector("p.hello strong");
-
+        private readonly By _editBillingAddress = By.XPath("//*[@id=\"top\"]/body/div/div[1]/div[2]/div/div[2]/div[2]/div/div[5]/div[2]/div[1]/div/div[1]/a");
+        private readonly By _firstNameField = By.Id("firstname"); 
+        private readonly By _middleNameField = By.Id("middlename");
```

Comment:
The selector for _editBillingAddress seems too long. I don't think you should have selectors on the title attribute. What if someone changes the "Telephone" label to just "Phone"?